src/Faithlife.WebRequests/Json/JsonWebServiceClientBase.cs

```diff
@@ -130,6 +155,27 @@ private Uri DoGetRequestUri(string relativeUriPattern, IEnumerable<KeyValuePair<
 			return uri;
 		}
 
+		private Uri DoGetRequestUri(string relativeUriPattern, params string[] uriParameters)
+		{
+			string uriText = JoinUriWithRelativePattern(m_baseUri, relativeUriPattern);
+
+			Uri uri = uriParameters != null && uriParameters.Length > 0 ? UriUtility.FromPattern(uriText, uriParameters) : new Uri(uriText);
```

Comment:
`&& uriParameters.Length > 0` seems like an intentional departure from line 151 but it isn't clear to me why it was needed.